src/Api/CertificatesApi.php

```diff
@@ -173,9 +173,7 @@ public function requestCertificate(
 
         $response = $this->client->post('/v2/ssl/certificates', $payload);
 
-        $headers = $response->headers();
-
-        return isset($headers['X-Process-Id']) ? (int) $headers['X-Process-Id'][0] : null;
+        return (int) $response->headers()['X-Process-Id'][0];
```

Comment:
I'm curious. What's the reason you chose to remove the `null` fallback here?
Technically if there's no process id being returned this would be `null`, right? If so, this will be transformed to `0` due to the typecasting.

Wouldn't you have a potentially invalid process id in that case?

Reply:
I know you mentioned in the description this header is always included according to RTR, but what if it's included but its value is `null`? Has this happened before and was this the reason why these methods originally allowed a `null` return type?

Reply:
a: not intended to return a null value from RTR